cmd/operator/main.go

```diff
@@ -89,6 +89,7 @@ func Main() int {
 	namespaceUserWorkload := flagset.String("namespace-user-workload", "openshift-user-workload-monitoring", "Namespace to deploy and manage user workload monitoring stack in.")
 	namespaceSelector := flagset.String("namespace-selector", "openshift.io/cluster-monitoring=true", "Selector for namespaces to monitor.")
 	configMapName := flagset.String("configmap", "cluster-monitoring-config", "ConfigMap name to configure the cluster monitoring stack.")
+	userWorkloadConfigMapName := flagset.String("userWorkloadConfigmap", "user-workload-monitoring-config", "ConfigMap name to configure the user workload monitoring stack.")
```

Comment:
why does this need to be configurable?

Reply:
Just followed pattern we had, see line above here, honestly happy to leave it out as well!

Reply:
my intuition is we should limit exposure here as we want to move to a CRD based setup :thinking: 

This cli flag increases the public API surfaces.

Reply:
good point, removed it!